crates/valence/src/instance.rs

```diff
@@ -28,6 +29,7 @@ use crate::view::ChunkPos;
 mod chunk;
 mod chunk_entry;
 mod paletted_container;
+pub mod weather;
```

Comment:
Keeping the weather module in the `instance` module doesn't really fit since it works on clients too. So it's original place is probably fine.